editor/animation_editor.cpp

```diff
@@ -1533,7 +1533,7 @@ void AnimationKeyEditor::_track_editor_draw() {
 			for (Map<SelectedKey, KeyInfo>::Element *E = selection.front(); E; E = E->next()) {
 
 				int idx = E->key().track;
-				int i = idx - v_scroll->get_value();
+				int i = idx - Math::floor(v_scroll->get_value());
```

Comment:
Is there a specific reason to use floor instead of round in this case?

Reply:
It has to use floor otherwise the offset is wrong, if the value is 0.5 etc. I have just changed the Math::floor to (int). Hope that is fine.